ESPController/platformio.ini

```diff
@@ -53,14 +54,10 @@ extra_scripts =
 upload_speed=921600
 upload_port=COM4
 
-; Use github version of the framework
-;platform_packages =
-;    platformio/framework-arduinoespressif32 @ https://github.com/espressif/arduino-esp32.git
-
-; SDK Version:v3.3.5-1-g85c43024c
 platform_packages =
-    platformio/framework-arduinoespressif32 @ 3.10006.210326
-
+   toolchain-xtensa32 @ ~2.80400.0
+   platformio/tool-esptoolpy @ ~1.30100
+   framework-arduinoespressif32 @ https://github.com/espressif/arduino-esp32.git#2.0.1
```

Comment:
The web interface 2.0.1 worked most of the time in incognito but failed when running normal mode for me, there is a known bug in 2.0.1 and 2.0.2 where the IDF httpd is artificially limited to 512b of headers. Switching to `https://github.com/espressif/arduino-esp32.git#esp32-s3-support` works in all envs from my testing which contains an update to the httpd to allow up to 1024b of headers (headers used were ~545b in my tests)

Reply:
Thanks for testing that. I manually deleted the old cookies from chrome and it worked on my setup. I'll follow the link you mentioned.

Reply:
I tried doing that myself but Chrome and other plugins are adding other headers which pushes it just barely over the limit

Reply:
But I'll keep this up on my setup and monitor it for a few weeks